src/App.css

```diff
@@ -35,18 +35,45 @@
   font-size: 1rem;
 }
 
-#chart-button {
+.chart-button {
   background-color: #C8F0E1;
   border: none;
   font-weight: bold;
-  text-align: left;
+  text-align: center;
   padding: 16px;
   padding-left: 10px;
   width: 100%;
   min-height: 100%;
   font-size: 1.25rem;
+
+
+  border-radius: 5px;
+  display: block;
+  margin: 10% auto 5%;
+  position: relative;
+}
+
+.chart-button::after {
+  content: '';
```

Comment:
Just a quick little animation that gives users a hint of the button being an expandable panel.